plugin/src/main/kotlin/com/facebook/testing/screenshot/build/PullScreenshotsTask.kt

```diff
@@ -44,6 +44,9 @@ open class PullScreenshotsTask : ScreenshotTask() {
   fun pullScreenshots() {
     val codeSource = ScreenshotsPlugin::class.java.protectionDomain.codeSource
     val jarFile = File(codeSource.location.toURI().path)
+    val outputDir = File(project.buildDir, "screenshots" + variant.name.capitalize())
```

Comment:
Lets move the creation of the `File` object to a reusable function so we dont have to keep the logic in sync between two files. I think it would fit fine in the companion object of `PullScreenshotsTask`